engine/src/main/java/org/terasology/rendering/nui/layers/mainMenu/UniverseSetupScreen.java

```diff
@@ -198,7 +198,9 @@ public void set(String value) {
         });
 
         WidgetUtil.trySubscribe(this, "addGenerator", button -> {
-            if (worldGenerator.getSelection().getUri().toString().equals("Core:heightMap")) {
+            //TODO: there should not be a reference from the engine to some module - the engine must be agnostic to what
+            //      modules may do
+            if (worldGenerator.getSelection().getUri().toString().equals("CoreWorlds:heightMap")) {
```

Comment:
I don't know how to deal with this, but as it is, that's a potential bug waiting around the corner... at least, there should be a generic way for modules to mark that some world gen is not suited for advanced universe setup

Reply:
Yeah, that whole project itself (multiworld) really needs to be resumed and finished. Along with the better phasing of things during game setup so you can get things like block registration and hot keys sorted out before you enter the world.